source/applications/basic/visualization/read_zdf_vis_3d.py

```diff
@@ -103,7 +103,7 @@ def _main():
     print(f"Reading {filename_zdf} point cloud")
     frame = zivid.Frame(filename_zdf)
 
-    point_cloud = frame.get_point_cloud().to_array()
+    point_cloud = frame.point_cloud().copy_data("xyzrgba")
     xyz = np.dstack([point_cloud["x"], point_cloud["y"], point_cloud["z"]])
     rgb = np.dstack([point_cloud["r"], point_cloud["g"], point_cloud["b"]])
```

Comment:
May I suggest using copy_data("xyz") and copy_data("rgba") separately?

Reply:
done